forum/views.py

```diff
@@ -14,6 +15,11 @@ class AnswerView(viewsets.ModelViewSet):
     serializer_class = AnswerSerializer
     permission_classes = [permissions.IsAuthenticated]
     
+    def list(self, request):
+        queryset = User.objects.all()
```

Comment:
The query set is already set correct i think 
you are setting queryset as all user object . its for answer list